statsmodels/sandbox/mice/mice.py

```diff
@@ -0,0 +1,1167 @@
+                 n_skip=10, init_args={}, fit_args={}):
+
+        if not isinstance(data, MICE_data):
+            raise ValueError("data argument must be an instance of MICE_data")
+        self.data = data
+        self.analysis_formula = analysis_formula
+
+        if not issubclass(analysis_class, statsmodels.base.model.Model):
+            raise ValueError("analysis_class must be a statsmodels model")
+        self.analysis_class = analysis_class
+
+        self.init_args = init_args
+        self.fit_args = fit_args
+        self.n_skip = n_skip
+
+    def __iter__(self):
+        return self
+
+    def next(self):
+
+        self.data.update_all(n_iter=self.n_skip + 1)
+
+        model = self.analysis_class.from_formula(self.analysis_formula,
+                                                 self.data.data,
+                                                 **self.init_args)
+        result = model.fit(**self.fit_args)
+
+        return result
+
+
+_mice_example_1 = """
+    >>> imp = mice.MICE_data(data)
+    >>> fml = 'y ~ x1 + x2 + x3 + x4'
+    >>> mice = mice.MICE_model(fml, sm.OLS, imp)
+    >>> mice.burnin(10)
+    >>> mice.run(10, 5)
+    >>> results = mice.combine()
+    >>> print results.summary()
+
+                              Results: MICE
+    =================================================================
+    Method:                    MICE       Sample size:           1000
+    Model:                     OLS        Scale                  1.00
+    Dependent variable:        y          Num. imputations       10
+    -----------------------------------------------------------------
+               Coef.  Std.Err.    t     P>|t|   [0.025  0.975]  FMI
+    -----------------------------------------------------------------
+    Intercept -0.0234   0.0318  -0.7345 0.4626 -0.0858  0.0390 0.0128
+    x1         1.0305   0.0578  17.8342 0.0000  0.9172  1.1437 0.0309
+    x2        -0.0134   0.0162  -0.8282 0.4076 -0.0451  0.0183 0.0236
+    x3        -1.0260   0.0328 -31.2706 0.0000 -1.0903 -0.9617 0.0169
+    x4        -0.0253   0.0336  -0.7520 0.4521 -0.0911  0.0406 0.0269
+    =================================================================
+"""
+
+class MICE(object):
+
+    __doc__ = """\
+    Use Multiple Imputation with Chained Equations to fit a model when
+    some data values are missing.
+
+    Parameters
+    ----------
+    model_formula : string
+        The model formula to be fit to the imputed data sets.
+    model_class : statsmodels model
+        The model to be fit to the imputed data sets.
+    data : MICE_data instance
+        MICE_data object containing the data set for which
+        missing values will be imputed
+
+    Examples
+    --------
+    Simple example using defaults:
+    %(mice_example_1)s
+    """ % {'mice_example_1' : _mice_example_1}
+
+    def __init__(self, model_formula, model_class, data,
+                 init_args={}, fit_args={}):
```

Comment:
same here: `init_kwds`, `fit_kwds`
in general it's better to avoid mutable default arguments, i.e. use `init_kwds=None` and set 
```
if init_kwds = None:
    init_kwds = {}
```
(I don't remember the details, just being conservative (defensive programming).)